persistence/sql/sqlsaga_required-in-some-versions_sqlpersistence_[,4).partial.md

```diff
@@ -0,0 +1 @@
+INFO: In order to use the normal `Saga<T>` class for sagas, upgrade to at least SQL Persistence 3.2.
```

Comment:
> the normal `Saga<T>` class

the NServiceBus Core `Saga<T>` class or the NServiceBus Core `Saga<T>` base class

Reply:
> upgrade to at least SQL Persistence 3.2.

upgrade SQL Persistence to Version 3.2 or above or use SQL Persistence Version 3.2 or above

Reply:
I made this change. I don't know why GitHub is still showing it this way!